bin/plotrapthor

```diff
@@ -490,9 +478,12 @@ def _calculate_screen(inscreen, residuals, pp, N_piercepoints, k, east, north, u
     D2 = np.sum(D**2, axis=2)
     C = -(D2 / r_0**2)**(beta_val / 2.0) / 2.0
     f = inscreen.reshape(N_piercepoints)
-    fitted_tec = np.dot(C, f) + residuals
     if is_phase:
-        fitted_tec = normalize_phase(fitted_tec)
+        fitted_vals = np.dot(C, f) + residuals
+    else:
+        fitted_vals = 10**(np.dot(C, f)) + residuals
```

Comment:
I don't know the context, so I might be completely wrong here, but shouldn't the base be `e`, instead of `10`?

Reply:
Yeah, it's good to be sure. In this case, it's due to the use of np.log10() in losoto where the screen is fit